.circleci/config.yml

```diff
@@ -486,3 +682,15 @@ workflows:
             branches:
               only:
                 - master
+      - benchmark_markdown_id_512
```

Comment:
This is just to confirm the new jobs still work, I will remove this as soon as it has.
Please do *not* merge if this is still here.

Reply:
Ok, ready to merge.